tests/test_makeTransmissionCurves.py

```diff
@@ -0,0 +1,122 @@
+#
+# LSST Data Management System
+#
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+from lsst.afw.geom import Point2D
+from lsst.obs.hsc import makeTransmissionCurves
+
+
+class MakeTransmissionCurvesTest(lsst.utils.tests.TestCase):
+
+    def testOptics(self):
+        wavelengths = np.linspace(4000, 10000, 100)
+        point = Point2D(1000, -500)
+        for curve in makeTransmissionCurves.getOpticsTransmission().values():
+            if curve is None:
+                return
```

Comment:
Don't you mean `continue`? Or are you taking advantage of some implicit ordering in the `dict`?

Reply:
I thought I was just making use of the fact that these dicts have only one or two elements, but now that I think about it that's bad, because I don't know the first element is the not-`None` one.  Will fix.